### DISCUSSION TITLE
Can the "deps" folder be deleted?

### DISCUSSION THREAD

**Discussion by jce-zz** [dani-garcia/vaultwarden#2066] [Q&A]
I realized that within the release folder of vaultwarden, there is a quite big folder called "deps". On my machine it has currently 1.1 GB. Can this folder be deleted safely or is it somehow required for running vaultwarden?

**Comment by RealOrangeOne** [comment]
The release binary should be (mostly) statically linked, so yes that should be fine to delete.

If you take a look at the `Dockefile`, the only file copied out of the build stage is the binary itself. So long as you have that and the vault files if needed, you should be good.

`target/release` is a staging area for Rust to put assets as it's building them, and potentially for related dynamic libraries etc. In this case, nothing else in there is necessary.
